charts/base-cluster/values.yaml

```diff
@@ -458,3 +463,8 @@ backup:
       limits:
         cpu: 500m
         memory: 500Mi
+
+weaveworks:
+  enabled: false
+  adminUser:
+    username: admin
```

Comment:
since you can only configure one user the path should change from `adminUser.username `and `admiUser.password` to just username, password